File: rafael/arrays/GeraradorDeGabarito.cs
class GeraradorDeGabarito 
{
	static void Main() 
	{
		System.Random gerador = new System.Random();
		int[] gabarito = new int[10];
		for(int i = 0; i < gabarito.Length; i++) 
		{
			gabarito[i] = (int)(gerador.NextDouble() * 3 + 1);
			System.Console.Write(gabarito[i] + " ");
		}
		System.Console.WriteLine("gabarito");
	}
}

File: rafael/arrays/GeraradorDeGabarito.java
class GeraradorDeGabarito {
	public static void main(String[] args) {
		int[] gabarito = new int[10];
		for(int i = 0; i < gabarito.length; i++) {
			gabarito[i] = (int)(Math.random() * 3 + 1);
			System.out.print(gabarito[i] + " ");
		}
		System.out.println("gabarito");
	}
}
